### PR TITLE
[tests-only][full-ci]Assign role using graph api 

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR adds a method to assign roles to user using graph api 

## Related Issue

- Fixes <https://github.com/owncloud/web/issues/8325>

## How Has This Been Tested?

- Locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by SagarGi** [comment]
This might be failing because the changes in `ocis-master` has not been backported to `stable-2.0` since the error is giving `404` response status code.

File: tests/e2e/cucumber/features/smoke/spaces/participantManagement.ocis.feature [hunk 1]
```diff
@@ -9,71 +9,71 @@ Feature: spaces participant management
...
-    When "Alice" changes the roles of the following users in the project space
-      | user  | role    |
-      | Carol | manager |
-    And "Alice" as project manager removes their own access to the project space
-    Then "Alice" should not be able to see space "team.1"
-    And "Alice" logs out
+#    And "Alice" creates the following project space using API
+#      | name | id     |
+#      | team | team.1 |
+#    When "Alice" logs in
```
Comment (saw-jan) [inline tests/e2e/cucumber/features/smoke/spaces/participantManagement.ocis.feature:15]:
```suggestion
    And "Alice" creates the following project space using API
      | name | id     |
      | team | team.1 |
    When "Alice" logs in
```

**Review by saw-jan — CHANGES_REQUESTED** [review]
?

**Review by kulmann — CHANGES_REQUESTED** [review]
With the `stable-6.0` branch (and the corresponding oCIS version) the graph api doesn't have `appRoleAssignments`. This change can only happen in web `master`, not in `stable-6.0`. If your other pull request is up to date you can close this one here.

**Pull request closed without merging**